src/test/groovy/com/capgemini/archaius/spring/SpringIncludeSystemPropertiesSpec.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2014 Capgemini (oss@capgemini.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.capgemini.archaius.spring
+
+import com.netflix.config.DynamicPropertyFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.io.ClassPathResource
+import org.springframework.core.io.Resource
+import org.springframework.core.io.support.PropertiesLoaderUtils
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import spock.lang.Specification
+
+/**
+ * @author Gayathri Thiyagarajan
```

Comment:
@sdwr98 d'you want to set your details here as the author (and add to the project README.md and other files appropriate)?